IFC4x3/Properties/n/NumberOfBoreholes_1DBeislK1CbfTPnPGztE29/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfBoreholes_1DBeislK1CbfTPnPGztE29" Name="NumberOfBoreholes" UniqueId="4d2e8b36-bd40-4c96-9759-c5943ddce089" PrimaryDataType="IfcInteger" />
```

Comment:
This is definitely already defined. Same proposal to rename to "BoreholeCount" and having the same type. Either `IfcInteger` or `IfcCountMeasure`.

Reply:
Done in [98fe0b2](https://github.com/bSI-InfraRoom/IFC-Specification/pull/678/commits/98fe0b26488006ded4b93c525a4779f670f1433a)